### PR TITLE
Pass reason with underlying client connection warnings.

### PR DESCRIPTION
**Pull request by mookerji**
Passes the message from a Requests connection exception, to be
reflected in the Console. Previously this context was dropped.

/cc @mfine

### CODE REVIEW

**Comment by mfine** [comment]
:+1:

**Comment by ljbade** [comment]
lets get this into release console as well as master

File: python/sbp/client/drivers/network_drivers.py [hunk 1]
```diff
@@ -209,15 +209,21 @@ def connect_write(self, source, whitelist, device_uid=None, pragma=None):
       gen = (msg.pack() for msg, _ in self.source)
       self.write_session.put(self.url, data=gen, headers=headers)
       self.write_response = True
-    except requests.exceptions.ConnectionError:
-      msg = "Invalid request to %s with headers %s." % (self.url, headers)
+    except requests.exceptions.ConnectionError as err:
```
Comment (ljbade) [inline python/sbp/client/drivers/network_drivers.py]: only tweak might be to put a space around the =?

**Comment by mookerji** [comment]
@ljbade yep, i'll take care of it.

**Comment by jkretzmer** [comment]
should a similar change also be made to the read calls?

**Comment by mookerji** [comment]
Eagle eye, @jkretzmer

**Pull request merged**